### PR TITLE
Use short urls for all php.net links

### PR DESCRIPTION
**Pull request by GromNaN**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | -

#### Summary

This pages detect the user language.

### CODE REVIEW

File: lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function __construct(Traversable $iterator)
         $this->storeCurrentItem();
     }
 
-    /** @see https://php.net/countable.count */
+    /** @see https://www.php.net/countable.count */
```
Comment (GromNaN) [inline lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php]: Do you agree this links are useless and should be removed?

Comment (malarzm) [reply lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php]: This one definitely :)

Comment (malarzm) [reply lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php]: Yeah, all in this file

Comment (GromNaN) [reply lib/Doctrine/ODM/MongoDB/Iterator/CachingIterator.php]: Done.

**Review by malarzm — APPROVED** [review]

**Pull request merged**